po/tr.po

```diff
@@ -18351,75 +17232,351 @@ msgid ""
+msgid "receiver:"
+msgstr "alıcı:"
+
+#: src/glossary.md:83
+msgid "reference counting:"
+msgstr ""
+
+#: src/glossary.md:84
+msgid "return:"
+msgstr ""
+
+#: src/glossary.md:85
+msgid "Rust:"
+msgstr "Rust:"
+
+#: src/glossary.md:86
+msgid ""
+"Rust Fundamentals:  \n"
+"Days 1 to 3 of this course."
+msgstr ""
+
+#: src/glossary.md:88
 msgid ""
-"```shell\n"
-"cargo run --bin server\n"
-"```"
+"Rust in Android:  \n"
+"See [Rust in Android](android.md)."
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:134
-msgid "and the client with:"
+#: src/glossary.md:90
+msgid "safe:"
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:136
-msgid ""
-"```shell\n"
-"cargo run --bin client\n"
-"```"
+#: src/glossary.md:91
+msgid "scope:"
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:142
-msgid "Implement the `handle_connection` function in `src/bin/server.rs`."
-msgstr ""
+#: src/glossary.md:92
+msgid "standard library:"
+msgstr "standart kütüphane:"
 
-#: src/exercises/concurrency/chat-app.md:143
-msgid ""
-"Hint: Use `tokio::select!` for concurrently performing two tasks in a "
-"continuous loop. One task receives messages from the client and broadcasts "
-"them. The other sends messages received by the server to the client."
+#: src/glossary.md:93
+msgid "static:"
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:146
-msgid "Complete the main function in `src/bin/client.rs`."
+#: src/glossary.md:94
+msgid "string:"
+msgstr "string:"
+
+#: src/glossary.md:95
+msgid "struct:"
+msgstr "yapı:"
+
+#: src/glossary.md:96
+msgid "test:"
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:147
-msgid ""
-"Hint: As before, use `tokio::select!` in a continuous loop for concurrently "
-"performing two tasks: (1) reading user messages from standard input and "
-"sending them to the server, and (2) receiving messages from the server, and "
-"displaying them for the user."
+#: src/glossary.md:97
+msgid "thread:"
+msgstr "iş parçacığı:"
+
+#: src/glossary.md:98
+msgid "thread safety:"
 msgstr ""
 
-#: src/exercises/concurrency/chat-app.md:151
-msgid ""
-"Optional: Once you are done, change the code to broadcast messages to all "
-"clients, but the sender of the message."
+#: src/glossary.md:99
+msgid "trait:"
+msgstr "özellik (trait):"
+
+#: src/glossary.md:100
+msgid "type:"
 msgstr ""
 
-#: src/thanks.md:3
-msgid ""
-"_Thank you for taking Comprehensive Rust 🦀!_ We hope you enjoyed it and "
-"that it was useful."
+#: src/glossary.md:101
+msgid "type inference:"
+msgstr "tür çıkarımı:"
+
+#: src/glossary.md:102
+msgid "undefined behavior:"
 msgstr ""
 
-#: src/thanks.md:6
-msgid ""
-"We've had a lot of fun putting the course together. The course is not "
-"perfect, so if you spotted any mistakes or have ideas for improvements, "
-"please get in [contact with us on GitHub](https://github.com/google/"
-"comprehensive-rust/discussions). We would love to hear from you."
+#: src/glossary.md:103
+msgid "union:"
+msgstr "birlik (union):"
```

Comment:
While "birlik" is not wrong as a translation at all, may be this can also be translated to Turkish as 
"Birlesimler" to further align the term with other translations of union, specifically with the Turkish translation of union of mathematical sets "Birlesim kumeleri" and with the translation of Union in SQL as referenced in [this Bilisim sozlugu dictionary](http://cayfer.bilkent.edu.tr/~cayfer/bilisim-sozlugu/tbd-ing-trk-sozluk.htm)

Reply:
In high-level fields such as SQL, the word union is translated as "Union". But when it comes to programming languages, the word union is translated as "unity". For this reason, I am in favor of keeping the translation of the word union as "unity". What do you think?

For example:
![image](https://github.com/google/comprehensive-rust/assets/12311762/581ff7e8-6664-45d5-82f7-1a1b3f1e00e6)

Reply:
@duyguisler ?